bld/CLMBuildNamelist.pm

```diff
@@ -3899,10 +3899,11 @@ sub setup_logic_rooting_profile {
 #-------------------------------------------------------------------------------
 
 sub setup_logic_friction_vel {
-  #
+  # Must be after canopyfluxes so that use_biomass_heat_storagge will be set
```

Comment:
use_biomass_heat_storagge -> use_biomass_heat_storage

Reply:
Got it. Thanks.